product.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./build/output.css">
+
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav class="nav ">
+
+        <div class="nav__logo">
+            <img src="img/finsweetlogo.svg" alt="">
+        </div>
+        <div class=" nav__menu ">
+            <div class="nav__list">
+                <a href="home.html" class="nav__item">Home</a>
+                <a href="product.html" class="nav__item">Product</a>
+                <a href="pricing.html" class="nav__item">Pricing</a>
+                <a href="#" class="nav__item">About Us</a>
+                <a href="#" class="nav__item">Blog</a>
+                <a href="#" class="nav__item">Contact</a>
+
+
+            </div>
+            <a href="#" class="button ">Free Trial</a>
```

Comment:
## prohibition of nested block
`button` is not a best practice to have a nested block